components/admin_console/reset_password_modal.jsx

```diff
@@ -141,7 +141,7 @@ export default class ResetPasswordModal extends React.Component {
                         >
                             <FormattedMessage
                                 id='admin.reset_password.close'
```

Comment:
You might consider changing the names ids of the translation strings to match the new text as well